app/services/idv/steps/verify_base_step.rb

```diff
@@ -199,13 +199,18 @@ def enqueue_job
 
         document_capture_session.requested_at = Time.zone.now
 
+        double_address_verification =
+          current_user.establishing_in_person_enrollment&.capture_secondary_id_enabled &&
+          flow_session['pii_from_user']['same_address_as_id'] == 'false'
```

Comment:
where does `same_address_as_id` get turned into a string? params?

Reply:
I _think_ it comes in from the [form](https://github.com/18F/identity-idp/blob/8d72e2070450849e01ab2018fad2eba87d2c7819/app/forms/idv/state_id_form.rb#L38-L43) that way

Would be great to parse booleans correctly. This has caused bugs before (not ones that made it to production, but still)